pontoon/base/static/js/locale_project.js

```diff
@@ -0,0 +1,65 @@
+$(function() {
+
+  // Search entire project with Enter (and remove default search handler)
+  $('.part .menu .search-project input[type=search]').off('keyup.search').keydown(function (e) {
```

Comment:
Also, semantically it's better to wrap the search field in a `<form>` element so that everything related to submitting the search is grouped together. Then, you can hook on the submit event of the form, which avoids running an event handler on every keypress and allows for submission of the form in ways besides hitting enter.